app/views/clients/_relatives_show.html.slim

```diff
@@ -4,7 +4,7 @@
       = rel.first_name
       = ' '
       = rel.last_name
-      = ', '
```

Comment:
Why did you remove the colon?
The colon was there, to make the difference sure between firstname and lastname.
Because there are always confusions if there are two firstname or two lastnames.